content/guides/kubernetes/kapp-gs.md

```diff
@@ -0,0 +1,259 @@
+      matchLabels:
+        app: spring-petclinic
+    replicas: 2
+    template:
+      metadata:
+        labels:
+          app: spring-petclinic
+      spec:
+        containers:
+        - name: spring-petclinic
+          image: springcommunity/spring-framework-petclinic
+          ports:
+          - name: http
+            containerPort: 8080
+  ---
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: spring-petclinic
+    namespace: spring-petclinic
+    labels:
+      app: spring-petclinic
+  spec:
+    ports:
+    - port: 80
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      app: spring-petclinic
+  EOF
+  ```
+
+1. Here is where  `kapp` gets involved. We will use it to create our deployment and service.
+```
+  kapp deploy -n kapp-demo -a spring-petclinic -f kapp-spring-petclinic.yaml
+  ```
+
+  This format is fairly similar to `kubectl apply -f kapp-spring-petclinic.yaml`, but it also has an application name. You can also provide a directory instead of a single file.
+
+  The next difference you will notice is it prompts you if you want to actually run the command. You can shortcut this by adding the `-y` flag. 
+
+  It should give you an output similar to the following before hit `y`.
+  ```
+  Target cluster 'https://35.247.3.14' (nodes: demo-default-pool-0f8526ab-rb7b, 2+)
+
+  Changes
+
+  Namespace         Name              Kind        Conds.  Age  Op      Op st.  Wait to    Rs  Ri
+  (cluster)         spring-petclinic  Namespace   -       -    create  -       reconcile  -   -  
+  spring-petclinic  spring-petclinic  Deployment  -       -    create  -       reconcile  -   -  
+  ^                 spring-petclinic  Service     -       -    create  -       reconcile  -   -  
+
+  Op:      3 create, 0 delete, 0 update, 0 noop
+  Wait to: 3 reconcile, 0 delete, 0 noop
+  ```
+  You can see that it’s creating the `spring-petclinic` namespace, deployment, and service inside that namespace.
+
+  Here is a more significant difference with `kubectl`. `kapp` will wait on the resources to become available before terminating and will also show the progress for each resource and tell you if it succeeded or failed. Don’t worry, I didn’t set you up for failure :). 
+
+1.  Now that we have the deployment and service up, we can verify it’s working properly. First we can just take a look and see that they are up.
+  ```
+  kubectl -n spring-petclinic get all
+  ```
+
+  And then we can create a container and curl the service from within it. We need to do this right now since the service type is `ClusterIP` (default) so we can’t see it from outside of the cluster itself.
+  ```
+  kubectl run -it --rm --restart=Never curl --image=curlimages/curl spring-petclinic.spring-petclinic.svc.cluster.local
+  ```
+### Looking At the App with kapp
+Okay, so let’s use kapp instead to take a look at our application. 
+
+1. We can take a look at everything we have running with kapp using the following command:
+  ```
+  kapp list
```

Comment:
Not sure if I've messed up something running through this, but I actually had to run `kapp list -n kapp-demo`, otherwise it was reporting that there were no apps in the `default` namesapce

Reply:
Yea, I forgot I changed this bit. Thanks